source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1480,6 +1481,20 @@ message TransformationReplaceLoadStoreWithCopyMemory {
   InstructionDescriptor store_instruction_descriptor = 2;
 }
 
+message TransformationReplaceOpSelectWithConditionalBranch {
+
+  // A transformation that takes an OpSelect instruction with a
+  // scalar boolean condition and replaces it with a conditional
+  // branch and an OpPhi instruction.
+
+  // The result id of the OpSelect instruction
+  uint32 select_id = 1;
+
+  // Fresh ids for the new blocks
+  uint32 true_block_fresh_id = 2;
+  uint32 merge_block_fresh_id = 3;
```

Comment:
I think we tend to have a comment per field and line of whitespace between field for protobuf messages, so I suggest commenting in a little more detail - e.g. explaining that in the true block there will be a copy object of the "then" id of the select, and that the merge block with choose between this and the "else" id of the select using an OpPhi.

Reply:
Done (after making the other changes)